### ISSUE TITLE
right click on marker

### ISSUE THREAD

**Issue by drakator** [FlowingCode/GoogleMapsAddon#67]
Hello, 
in version 1.6.0
I can't get right click event over marker 
(left click is working, but does not give me clickCounts, ctrl, shift, and alt keys)

best regards

**Comment by paodb** [comment]
Hello, currently the component doesn't have an API to handle right click on markers but the web-component has the event for that (_google-map-marker-rightclick_) so you could get the right click working with something like this:

```
marker.getElement().setProperty("clickEvents", true);
marker.getElement().setProperty("clickable", true);
marker.getElement().addEventListener("google-map-marker-rightclick", event -> {
   // do something here         
}).addEventData("event");
````
The only thing is that this returns a MapMouseEvent and it contains only two fields: [domEvent & latLng](https://developers.google.com/maps/documentation/javascript/reference/map#MapMouseEvent).

**Comment by paodb** [comment]
I've investigated this a little bit more and I run into [this example](https://jsfiddle.net/tqj3aug8/3/) that I modified a little bit and shows that is possible to add a native **rightclick** **listener** to a google map's marker and obtain some data like altKey, ctrlKey, shiftKey and metaKey (for Mac), Window-relative coordinates: clientX/clientY and document-relative coordinates: pageX/pageY. I think that we can implement something similar in our google-maps web component that is the base of this addon. Not sure if it's possible to obtain clickCounts.

![google-maps-marker](https://user-images.githubusercontent.com/30666649/169608148-3604eb6b-ff69-4475-831d-5a01bb37e5dd.gif)
